src/templates/toolkit/bet/static.html.tt

```diff
@@ -50,7 +50,7 @@
                         </div>
                         <div class="big-col">
                            <div id="expiry_type_duration">
-                              <input id="duration_amount" type="number" min="1" max="366" class="small_width_input" /><label></label>
```

Comment:
As discussed with Ashkan, we are only showing one error everywhere not multiple errors. The backend already replies with errors, if any, and is displayed in the frontend!